settings.php

```diff
@@ -330,3 +329,17 @@
  * built for Backdrop.
  */
 $settings['backdrop_drupal_compatibility'] = TRUE;
+
+
+/**
+ * Include a local settings file.
+ *
+ * To make local development easier, you can add a file that contains settings
+ * for CSS/JS aggregation, caching, or anything else commonly changed during
```

Comment:
This documentation is currently incorrect. You can't disable CSS/JS aggregation through settings.php currently. It's config, not a `$settings` option.